queries.sql

```diff
@@ -5,4 +5,61 @@ SELECT date_of_birth FROM animals WHERE name = 'Agumon' OR name = 'Pikachu';
 SELECT name escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = TRUE;
 SELECT * FROM animals WHERE name <> 'Gabumon';
-SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
\ No newline at end of file
+SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
+
+--UPDATE SPECIES COLUMN
+
+BEGIN;
+UPDATE animals
+SET species = 'digimon'
+WHERE name LIKE '%mon';
+
+UPDATE animals
+SET species = 'pokemon'
+WHERE species IS NULL OR species = '';
+SELECT * FROM animals;
+
+COMMIT;
+SELECT * FROM animals;
+
+--DELETE 2022 BORN
+
+BEGIN;
+
+DELETE FROM animals
+WHERE date_of_birth  > '2022-01-01';
+
+SAVEPOINT first_save;
+
+UPDATE animals
+SET weight_kg  = weight_kg  * -1;
+
+ROLLBACK TO SAVEPOINT first_save;
+
+UPDATE animals
+SET weight_kg = weight_kg * -1
+WHERE weight_kg < 0;
+
+COMMIT;
+SELECT * FROM animals;
```

Comment:
- [ ] Before the transaction highlighted in the code snippet, you ought to write a transaction that deletes all data in the `animals` table and then roll it back. In the process of the transaction, you ought to verify that all data were deleted from the `animals` table. Kindly carry out this transaction as it is a requirement. 👍 💻 

![image](https://github.com/otmaneechchafyky/vet_linic_database/assets/77106962/8a00c2c1-5749-4c19-b4d5-8731c0bcd32d)

### N:B: You have the screenshot attached, but the code is not written in this file. Kindly write the code for it.  😎